pkg/kubeclient/banneduser.go

```diff
@@ -2,7 +2,7 @@ package kubeclient
 
 import (
 	"context"
-	"crypto/md5"
+	"crypto/md5" //nolint:gosec
```

Comment:
just out of curiosity - what is the error we ignore here?

Reply:
md5 is a weak crypto algorithm, but since already use it, we would need to update all our resources if we decide to use something else, such as sha-1 for example.